kuzzleio/kourou
# Fix issue with scroll duration & improve flag

## What does this PR do?




* Fix issue where scrollId parameters was not taken into account
* Improve migrate command support with dry-run option, regex pattern and specific scroll duration for large collection support

### How should this be manually tested?


  - Step 1 : Start two elasticsearch
  - Step 2 : Run this command `./bin/run es:migrate --src http://localhost:9201 --dest http://localhost:9200 --dry-run --pattern i-want-to-keep-this`
  - Step 3 :  See command output
  ...

### Other changes



### Boyscout
